lib/improver/utilities/temporal.py

```diff
@@ -182,3 +190,179 @@ def forecast_period_coord(
            "coordinate were also not available for calculating "
            "the forecast_period.".format(cube))
     raise CoordinateNotFoundError(msg)
+
+
+def iris_time_to_datetime(time):
```

Comment:
Have you not already used time as
```
from datetime import time
```

Reply:
Certainly confusing I've changed this  to dt_time